src/scantailor/core/zones.py

```
@@ -0,0 +1,285 @@
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import TYPE_CHECKING
+
+import numpy as np
+from pydantic import BaseModel
+
+if TYPE_CHECKING:
+    from numpy.typing import NDArray
+
+
+class ZoneCategory(str, Enum):
```

Ruff (0.15.0)

[warning] Class ZoneCategory inherits from both `str` and `enum.Enum`

Inherit from `enum.StrEnum`

(UP042)